data/categories.txt

```diff
@@ -5159,7 +5159,8 @@ zh-Hans:摩托车租赁
 zh-Hant:摩托車租賃
 
 amenity-cinema|@category_entertainment
-en:3Cinema|U+1F3A6|U+1F3AC
+en:3Cinema|Film|Movie Theatre|U+1F3A6|U+1F3AC|
+en-US:Movie Theater
```

Comment:
Not sure if en-US will work in such way. Did you test this change with different system languages? List all variants in en: ?

@vng